lib/root.js

```diff
@@ -15,6 +15,14 @@ process.on('exit', function (code) {
     process.exit(1)
 })
 
+// need to autoend if a teardown is added.
+// otherwise we may never see process.on('exit')!
+tap.tearDown = tap.afterAll = function (fn) {
+  var ret = Test.prototype.afterAll.apply(this, arguments)
+  this.autoend()
```

Comment:
I guess it's not *that* weird, since tap.test !== t.test, but maybe it ought to be pre-bound like those other methods?